pkg/types/aws/validation/platform.go

```diff
@@ -54,7 +54,10 @@ var (
 func ValidatePlatform(p *aws.Platform, fldPath *field.Path) field.ErrorList {
 	allErrs := field.ErrorList{}
 	if _, ok := Regions[p.Region]; !ok {
-		allErrs = append(allErrs, field.NotSupported(fldPath.Child("region"), p.Region, validRegionValues))
+		// TODO remove this when AWS China regions are officially supported
+		if p.Region != "cn-north-1" && p.Region != "cn-northwest-1" {
+			allErrs = append(allErrs, field.NotSupported(fldPath.Child("region"), p.Region, validRegionValues))
+		}
```

Comment:
I think we need to add validation that AMI ID is specified when these regions are used.

Reply:
Do we need validate here? it looks like it will fail without setting. amiID with errors:

```
FATAL failed to fetch Terraform Variables: failed to fetch dependency of "Terraform Variables": failed to generate asset "Image": no RHCOS AMIs found in cn-northwest-1
```